src/dataprep/pipeline.sh

```diff
@@ -72,14 +75,18 @@ Rscript -e "rmarkdown::render('$script_path/reports/quality_mag.Rmd', output_dir
 Rscript -e "rmarkdown::render('$script_path/reports/quality_proquest.Rmd', output_dir = '$output_path')" \
     &> $logfile_path/quality_proquest.log
 
+Rscript -e "rmarkdown::render('$script_path/reports/sample_size_linking.Rmd', output_dir = '$output_path')" \
+    &> $logfile_path/sample_size_linking.log
 
 # ## Link graduates
 bash $script_path/link/graduates.sh $logfile_path
 
 Rscript -e "rmarkdown::render('$script_path/reports/quality_linking.Rmd', output_dir = '$output_path')" \
     &> $logfile_path/quality_linking.log
 
-python -m $script_path.link.prep_linked_data &> $logfile_path/prep_linked_data.log
+python -m $script_path.link.prep_linked_data \
```

Comment:
this should only be called after all the linking (and writing to csv) is done as it relies on the most recent links